src/Microsoft.Diagnostics.Monitoring.StartupHook/Exceptions/Pipeline/ExceptionPipelineExceptionContext.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.StartupHook.Exceptions.Pipeline
+{
+    internal sealed class ExceptionPipelineExceptionContext
```

Comment:
From a design perspective, what's the purpose of this class as an abstraction around `DateTime`?

Reply:
For now, it's just the timestamp. But I made it a separate class so that way if we add more contextual information in the future (e.g. first chance vs unhandled) then we don't need to refactor a bunch of the code to add yet another property.